.github/ISSUE_TEMPLATE/java-stable-release-template.md

```diff
@@ -0,0 +1,68 @@
+### RELEASE_VERSION_NUMBER-alpha.1 pre-release checklist 
+
+- [ ] Update the SNAPSHOT version in this repo's `README.md` file 
+- [ ] Change version name to `RELEASE_VERSION_NUMBER-alpha.1` in this repo's `gradle.properties` (make sure to remove `-SNAPSHOT`)
+- [ ] Create a new release in Github with a release title of `Mapbox Java SDK vRELEASE_VERSION_NUMBER-alpha.1` and a tag version of `vRELEASE_VERSION_NUMBER-alpha.1`. This will publish artifacts to Bintray. 
+
+<details>
+<summary>What to do in Bintray</summary>
+After the tag is created, the CI build starts automatically at https://circleci.com/gh/mapbox/mapbox-java and our CircleCI circle.yml file instructs CircleCI to push the artifact to Bintray.
+
+Once CircleCI successfully pushes the artifact files to Bintray, log into Bintray with the Mapbox credentials. Once you're signed in, visit this release numbers's `Files` page for each of the Java SDK modules:
+
+- https://bintray.com/mapbox/mapbox/mapbox-sdk-turf#files/com/mapbox/mapboxsdk/mapbox-sdk-turf
+
+- https://bintray.com/mapbox/mapbox/mapbox-sdk-geojson#files/com/mapbox/mapboxsdk/mapbox-sdk-geojson
+
+- https://bintray.com/mapbox/mapbox/mapbox-sdk-services#files/com/mapbox/mapboxsdk/mapbox-sdk-services
+
+- https://bintray.com/mapbox/mapbox/mapbox-sdk-core#files/com/mapbox/mapboxsdk/mapbox-sdk-core
+
+Verify the attached files for each module, especially the POM which contains the artifact's dependencies. Go to the `Maven Central` tab and click the `Sync` button to sync the artifact with the repository.
+</details>
+
+- [ ] Change version name back to having `-SNAPSHOT` at the end
+- [ ] Update this repo's `CHANGELOG.md`
```

Comment:
This could be the first step so that the git commit being tagged contains an up-to-date changelog.

Reply:
Fixed in my latest commit